src/app/(private)/(routes)/(member)/forum/[postId]/[...titleSlug]/page.tsx

```diff
@@ -25,41 +25,34 @@ type PostPageProps = {
   }
 }
 
-// export async function getServerSideProps({ params }: PostPageProps) {
-//   const post = await getPost(params.postId)
-//   const parsedContent = JSON.parse(post?.content as string)
-//   const description =
-//     parsedContent.content[0]?.content[0]?.text || 'Default description'
-
-//   const metadata: Metadata = {
-//     title: post?.title || '',
-//     description: description || '',
-//     openGraph: {
-//       title: post?.title || '',
-//       description: description || '',
-//       url: `https://borderless-community-test.vercel.app/forum/${params.postId}/${params.titleSlug}`,
-//       type: 'article'
-//     }
-//   }
-
-//   return {
-//     props: {
-//       post,
-//       metadata
-//     }
-//   }
-// }
-
 export async function generateMetadata({
   params
 }: PostPageProps): Promise<Metadata> {
   const post = await getPost(params.postId)
   const parsedContent = JSON.parse(post?.content as string)
-  const description = parsedContent.content[0].content[0].text
+  const description =
+    parsedContent.content?.[0]?.content?.[0]?.text || 'Default description'
+  const img = parsedContent.content?.[1]?.attrs?.src || appLogo.src
+  const title = post?.title || 'Default Title'
+  const altText = `Image for ${title}`
 
   const metadata: Metadata = {
-    title: post?.title,
-    description
+    title,
+    description,
+    openGraph: {
+      title: post?.title || 'Title',
+      description: description || 'description',
```

Comment:
Could we consider setting the default value to an empty string? I believe it might be preferable to displaying 'Title' by default.